Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -46,8 +46,6 @@ - (instancetype)initWithFrame:(CGRect)frame
     _placeholderView.numberOfLines = 0;
     [self addSubview:_placeholderView];
 #else // [TODO(macOS GH#774)
-    NSTextCheckingTypes checkingTypes = 0;
```

Comment:
do we know the history why someone added this?

Reply:
Nope. Change was made before React Native macOS was open source :(